mapillary_tools/process_video.py

```diff
@@ -35,6 +37,7 @@ def timestamps_from_filename(full_image_list,
 
 def sample_video(video_file,
                  import_path,
+                 geotag_source,
```

Comment:
required arg geotag_source might break calls for this func , sample_video command in tools does not have this arg